RFC-010-core-strategy/core-strategy.md

```diff
@@ -0,0 +1,67 @@
+# Executive Summary
```

Comment:
I recommend an author/published section to start, for accountability and clear indication of age. (Slightly related to my comment at the end about a changelog)

Reply:
Got this in the new document history section.